odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -197,3 +197,23 @@ Stopping ODK-X Sync Endpoint
 
       $ docker volume rm (docker volume ls -f "label=com.docker.stack.namespace=syncldap" -q)
 
+.. _sync-anonymous:
+
+Anonymous Access for ODK-X Sync Endpoint
+-----------------------------------------
+
+Navigate to :file:`sync-endpoint-default-setup/config/sync-endpoint/security.properties` 
+
+  - To Enable Anonymous access set the following fields to *true*
```

Comment:
Could you add information on how to check if anonymous access is enabled? You can check it by going to either `web-ui/admin/users` or `odktables/${APP_NAME}/usersInfo`.